lib/game/board.rb

```diff
@@ -0,0 +1,52 @@
+require 'figure/knight'
+require 'figure/pawn'
+require 'figure/king'
+require 'figure/bishop'
+require 'figure/queen'
+require 'figure/rook'
+require 'figure/base'
+require 'support/moves_helper'
+module Game
+  class Board
+    include Support::MovesHelper
+    attr_accessor :chess_board
+
+    def initialize
+      @chess_board = {}
+      start_statement_white
+      start_statement_black
+    end
+
+    def start_statement_white
+      @chess_board[[1, 1]] = Figure::Rook.new([1, 1], 'white')
```

Comment:
please look at solution below:

```ruby
def figure_positions
  {
    Figure::Rook => [[1, 1], [8, 1],  
    Figure::Knight => [[2, 1], [7, 1]],
    Figure::Bishop => [[3, 1], [6, 1]],
    Figure::Queen => [[4, 1]],
    Figure::King => [[5, 1]]
    Figure::Pawn => (1..8).map {|i| [i, 2] }
  }
end

def start_statement_white
  figure_positions.each do |figure_class, positions|
    positions.each do |position|
      @chess_board[position] = figure_class.new(position, 'white')
    end 
  end
end
```

do you see what you can do next with that? :-)